packages/ERTP/src/issuer.js

```diff
@@ -44,7 +44,7 @@ function produceIssuer(allegedName, mathHelpersName = 'nat') {
         const srcPayment = E.unwrap(paymentP);
         const srcPaymentBalance = paymentLedger.get(srcPayment);
         assertAmountEqual(srcPaymentBalance, optAmount);
-        const purseBalance = purse.getBalance();
+        const purseBalance = purse.getAmount();
```

Comment:
If you're going to change in this direction, please use `getCurrentAmount` rather than `getAmount` for changing amounts. Most of our remotely invoked `getFoo()` methods give stable answers, or give stable answers until irreversibly over. Let's find the ones that instead give changing answers over time. Once we do, we might decide it's better to add "Current" there rather than losing it here.

Looking at all the query method declarations in issuer in this branch
   * Brand
      * isMyIssuer stable
      * allegedName stable
   * Issuer
      * getBrand stable
      * allegedName stable
      * getAmountMath stable
      * getMathHelpersName stable
      * isLive stable until over, which is the point
      * getAmountOf (needs correction in chainmail) stable until over
   * Mint
      * getIssuer stable
   * Purse
      * allegedBrand (chainmail still says issuer) stable
      * ***getBalance changing***, the controversy
   * Payment
      * allegedBrand (need correction in chainmail) stable

Of these, `purse.getBalance()` is the only one designed to give different meaningful answers over time. 

Yesterday's discussion about the necessary hazards of asynchrony emphasizes that we should do more to alert about the difference between stable and changing quantities.

Rather than "Current", we might even consider "Recent". It is not technically correct as a name at the object receiving the message, because its answer is accurate at that moment. But it is more accurate as to how a remote requestor should understand the answer.

Reply:
Btw, curious about why most of these are `getFoo()` but `allegedBrand()` alone is not. Either is plausible, but we should establish a convention and stick with it.

Perhaps we could address both issues at the same time. Drop the `get` prefix from all the stable or stable-until-over ones. Use a `current` prefix on the changing one(s). So `issuer.amountOf(payment)` and `purse.currentAmount()`. I am not suggesting that you do any such global naming reform in the PR, but want to start the discussion.

Reply:
Yeah, I like the idea of "RecentAmount" for the balance that might change. 

Re: `getFoo` vs `allegedBrand` - generally I prefer a `get` prefix for clarity. Removing it in `allegedBrand` was a suggestion by Dean, I think. I'd prefer to add it back in if we need consistency.

Reply:
@dtribble , do you agree that it should either be , for example, `amount()` and `allegedBrand()` or `getAmount()` and `getAllegedBrand()`? That we should pick a convention and then stick with it consistently? Or is the idea that `alleged` be a reused prefix like `get` is? Otherwise, I don't see what's special about `allegedBrand()`.

@katelynsills we do not need to settle this in this PR.

Reply:
I changed to `getAllegedBrand` and `getAllegedName` for consistency